Sources/SentryCrash/Recording/SentryCrashC.c

```diff
@@ -112,11 +113,20 @@ onCrash(struct SentryCrash_MonitorContext *monitorContext)
     // because we gonna call into non async-signal safe code
     // but since the app is already in a crash state we don't
     // mind if this approach crashes.
-    if (g_saveScreenShot) {
-        char crashScreenshotsPath[SentryCrashCRS_MAX_PATH_LENGTH];
-        sentrycrashcrs_getScreenshotsPath_forReport(
-            g_lastCrashReportFilePath, crashScreenshotsPath);
-        g_saveScreenShot(crashScreenshotsPath);
+    if (g_saveScreenShot || g_saveViewHierarchy) {
+        char crashAttachmentsPath[SentryCrashCRS_MAX_PATH_LENGTH];
+        sentrycrashcrs_getAttachmentsPath_forReport(
+            g_lastCrashReportFilePath, crashAttachmentsPath);
+
```

Comment:
h: You need to make sure the folder exists or create it here

Reply:
See one line below: `if (sentrycrashfu_makePath(crashAttachmentsPath)) {`